backbone/src/main/java/org/researchstack/backbone/model/ConsentQuizModel.java

```diff
@@ -18,6 +18,10 @@
     private String incorrectIcon = "rsb_quiz_retry";
     private String correctIcon   = "rss_ic_quiz_valid";
 
+    ConsentQuizModel() {
```

Comment:
Yes and also because Model objects should at least have package level default constructors.